tests/helpers.py

```diff
@@ -0,0 +1,30 @@
+"""Checks that the cookiecutter works."""
+
+import pathlib
+import subprocess
+
+
+def gen_package(
```

Comment:
I'd personally prefer `generate_package` as I think abbreviations in variable / function names make it more difficult to remember which particular shortening is being used and also less explicit to someone who might not understand what `gen` means.